zallet/src/components/keystore.rs

```diff
@@ -505,6 +505,7 @@ impl KeyStore {
         Ok(seed_fp)
     }
 
+    #[allow(dead_code)]
     pub(crate) async fn encrypt_and_store_legacy_seed(
```

Comment:
I NACKed this change in #109 because at the time I thought that what is now #152 would be merged shortly, meaning that this change (and the other it proposed) was creating unnecessary work. Ironically we are now in a position where #152 _is_ likely to be merged shortly, but I'm now less averse to this change as long as we do indeed remember to do the additional work to revert it once #152 merges.